src/LaravelMapsServiceProvider.php

```diff
@@ -27,9 +27,9 @@ public function boot()
             ], 'config');
 
             // Publishing the views.
-            /*$this->publishes([
-                __DIR__.'/../resources/views' => resource_path('views/vendor/laravel-maps'),
-            ], 'views');*/
+            $this->publishes([
+                __DIR__.'/../resources/views/components' => resource_path('views/vendor/maps/components'),
```

Comment:
Maybe change this:
resource_path('views/vendor/maps/components')
to this to be more explicit:
resource_path('views/vendor/larswiegers/laravel-maps/components')

Reply:
@LarsWiegers I am not sure that it will publish out to that more explicit directory since your package is referred to as `maps` in the service provider. I think I tried that at one point or another but it required many more changes than I though you might accept.

Ill leave that up to you, I can make the change to the more explicit path but that will require more changes within the service provider than already in this PR.

Reply:
Ah okay, than this is fine. Merging!